README.md

```diff
@@ -681,3 +437,120 @@ thereafter it should be possible to use one of the different checkpatch targets
 in the [Makefile](Makefile). There are targets for checking all files, checking
 against latest commit, against a certain base-commit etc. For the details, read
 the [Makefile](Makefile).
+
+## 6. repo manifests
```

Comment:
What we should avoid is to duplicate our information. In the past we have tried to have all our documentation in `optee_os`, so I still think it's a good idea to put the information there (optee_os) and then link from other gits to point at this particular git instead.

One caveat! For the GP conference in October, the URL to `https://github.com/OP-TEE/manifest` has been sent out to the attendees, so if we cleanup the manifest git, we must temporary add some instructions for the GP conference.